<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Changelog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../images/openshot.svg</normaloff>../../images/openshot.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabChangelog">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_openshot_qt">
      <attribute name="title">
       <string notr="true">openshot-qt</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="vbox_openshot_qt">
         <item>
          <layout class="QHBoxLayout" name="hbox_openshot_qt">
           <item>
            <widget class="QLineEdit" name="txtChangeLogFilter_openshot_qt">
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>Filter Changelog</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_libopenshot">
      <attribute name="title">
       <string notr="true">libopenshot</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="vbox_libopenshot">
         <item>
          <layout class="QHBoxLayout" name="hbox_libopenshot">
           <item>
            <widget class="QLineEdit" name="txtChangeLogFilter_libopenshot">
             <property name="placeholderText">
              <string>Filter Changelog</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_libopenshot_audio">
      <attribute name="title">
       <string notr="true">libopenshot-audio</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="vbox_libopenshot_audio">
         <item>
          <layout class="QHBoxLayout" name="hbox_libopenshot_audio">
           <item>
            <widget class="QLineEdit" name="txtChangeLogFilter_libopenshot_audio">
             <property name="placeholderText">
              <string>Filter Changelog</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblGitHubLink">
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>258</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnclose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnclose</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
